RELEASES.md

```diff
@@ -1,3 +1,436 @@
+# Version 3.0 (2020-10-09)
+
+Be prepared for the ink! 3.0 release notes because the whole version was basically a rewrite of
+all the major components that make up ink!. With our experience gained by the last released versions
+of ink! we were able to detect weak spots of the design and provided ink! with more tools,
+more features and more efficiency as ever. Read more below ...
```

Comment:
```suggestion
more features and more efficiency as ever. Read more below…
```

Reply:
is that how you actually have to do it according to typists?

Reply:
Yup: https://en.wikipedia.org/wiki/Ellipsis

Reply:
I mean it's really just a nitpick, but for me in the same class as using "->" vs "➝", so yeah just decide for yourself ;-).

Reply:
Advises a space between "below` and the three dots though: Read more below …

Reply:
done, thanks